### ISSUE TITLE
[Bug]: Pipewire per-application volume of Floorp gets reset after a while

### ISSUE THREAD

**Issue by RandomLegend** [Floorp-Projects/Floorp#1106]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have tried to create a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

In pipewire you can adjust per-application volumes. I use a [deej](https://github.com/omriharel/deej) and that utilizes it for example.

If i set the application volume of floorp to 50% for example and pause a youtube video, it will reset back to 100% after a few seconds of pause. Also if you pause a video and immediately start a new one (or autoplay is activated) it will also revert to 100%. This of course is not specific to youtube but there i can trigger this effect the quickest.

This issue occurs also on Firefox and this is a [known issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1524849) for many years so i know this issue isn't on your side. But mozilla appears to have absolutely zero intentions to fix this, so i hope maybe you can find a solution for this in your fork...

This is literally the only reason i still use Chromium based browsers, as they have this fixed. I use the volume control all the time as it's very important for me.

### Reproduction Steps

1. Start a audio and / or video playback
2. Adjust the application volume via pipewire (also affects windows afaik) to an arbitrary percentage
3. Pause the video for a few seconds
4. Unpause the video
5. Volume reverted back to 100%

### Expected Behavior

1. Start a audio and / or video playback
2. Adjust the application volume via pipewire (also affects windows afaik) to an arbitrary percentage
3. Pause the video for a few seconds
4. Unpause the video
5. Volume stays the same as before the pause

### Actual Behavior

1. Start a audio and / or video playback
2. Adjust the application volume via pipewire (also affects windows afaik) to an arbitrary percentage
3. Pause the video for a few seconds
4. Unpause the video
5. Volume reverted back to 100%

### Screenshots

https://github.com/Floorp-Projects/Floorp/assets/38812603/c8c7e164-db00-4450-88d6-13f739ba745f

### Browser Version

11.12.0 (64-Bit)

### Operating System

Garuda Linux

### Additional Context

This bug also occurs on forks of your project like FireDragon

**Comment by RandomLegend** [comment]
@surapunoyousei as far as i know this is very reproducible. May i ask what OS you are on and if it really works for you?

**Comment by surapunoyousei** [comment]
Win  11

**Comment by RandomLegend** [comment]
And for Win11 it works?

**Comment by Sasikuttan2163** [comment]
I'm seeing the same issue on pipewire. @surapunoyousei can you change the tag to reproducible?

**Comment by RandomLegend** [comment]
@Sasikuttan2163 yeah in my video it jumped to 83% after opening a new video, but it jumps to 100% after pause & unpause

In all the mozilla bugreports and forum entries i found the numbers are wildly different aswell. Some say 79% some 81%

**Comment by RandomLegend** [comment]
Any news on this?
